### PR TITLE
Describe Ts4231 sample rate

### LINKED ISSUE #87: Specify Ts4231 sample rate
Do that on hs64 tutorial page

### PR DESCRIPTION
**Pull request by cjsha**
Resolve #87

### CODE REVIEW

File: articles/hardware/hs64/ts4231.md [hunk 1]
```diff
@@ -3,10 +3,11 @@ uid: hs64_ts4231
...
\ No newline at end of file
+The <xref:OpenEphys.Onix1.TS4231V1PositionData> operator generates a sequence of [TS4231V1PositionDataFrames](xref:OpenEphys.Onix1.TS4231V1PositionDataFrame). `TS4231V1PositionData` emits a `TS4231V1PositionDataFrame` when a receiver on the headstage successfully receives a coordinated set of four IR sweeps - two from each lighthouse. These coordinated sweeps occur at a rate of 30 Hz, so the maximum achievable sample rate is 30 Hz. In practice, the sample rate depends on the conditions of the behavioral arena. Achieving a 30 Hz sample rate requires that that the receiver is within range of the lighthouses and that there is no occlusion between the receiver and the lighthouses. There are four receivers on the headstage64 to increase the likelihood that at least one of them successfully receiving a complete lighthouse signal. Each receiver that successfully receives the full lighthouse signal results in `TS4231V1PositionData` emitting a `TS4231V1PositionDataFrame`. This means it is possible for the `TS4231V1PositionData` operator to emit multiple `TS4231V1PositionDataFrames` for a given set of coordinated sweeps.
```
Comment (jonnew) [inline articles/hardware/hs64/ts4231.md:12]: IR: Infrared
sweeps: what are they? why are they relevant? If you are going to provide this level of detail, you need to setup what you are talking about.

Diagram would be good with call outs.

**Review by jonnew — CHANGES_REQUESTED** [review]

File: articles/hardware/hs64/ts4231.md (hunk 1 above)
Comment (cjsha) [reply articles/hardware/hs64/ts4231.md]: I think you're right. I'm going to limit this description to as brief of an overview as I think is necessary. Instead, we should link out to the hardware docs if users want those details. If my latest commit looks good to you, please resolve conversation

File: articles/hardware/hs64/overview.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ These are the devices available on the Headstage64:
     - Optional adjustable digital high-pass filter with cutoff configurable from 0.146 Hz to 3309 Hz
     - Three auxiliary ADC channels sampled at 30 kHz with 16 bit depth
 - [Bno055](xref:hs64_bno055): 9-axis IMU for real-time, 3D orientation tracking sampled up to ~100 Hz for easy automated commutation with Open Ephys commutators
-- [Ts4231](xref:hs64_ts4231): For compatibility with HTC Vive Lighthouses for real-time, 3D position tracking
+- [Ts4231](xref:hs64_ts4231): For compatibility with HTC Vive Lighthouses for real-time, 3D position tracking sampled at 30 Hz in ideal conditions
```
Comment (jonnew) [inline articles/hardware/hs64/overview.md:16]: 3x HTC Vive Lighthouse receivers for real-time, 3D position tracking sampled at 30 Hz per reciever in ideal conditions

File: articles/hardware/hs64/ts4231.md [hunk 3]
```diff
@@ -3,10 +3,28 @@ uid: hs64_ts4231
...
-::: workflow
+::: workflow 
 ![/workflows/hardware/hs64/ts4231.bonsai workflow](../../../workflows/hardware/hs64/ts4231.bonsai)
 :::
 
-The <xref:OpenEphys.Onix1.TS4231V1PositionData> operator generates a sequence of <xref:OpenEphys.Onix1.TS4231V1PositionDataFrame>s.
\ No newline at end of file
+The <xref:OpenEphys.Onix1.TS4231V1PositionData> operator generates a sequence of
+[TS4231V1PositionDataFrames](xref:OpenEphys.Onix1.TS4231V1PositionDataFrame). These `TS4231V1PositionDataFrames` are
+emitted when a receiver on the Headstage64 successfully receives a fully formed set of signals from both lighthouses.
```
Comment (jonnew) [inline articles/hardware/hs64/ts4231.md:15]: fully formed set of ssignals from both lighthouses. -> a sequence of 6 optical signals from each lighthouse.

File: articles/hardware/hs64/ts4231.md [hunk 4]
```diff
@@ -3,10 +3,28 @@ uid: hs64_ts4231
...
+With no occlusions, the maximum achievable sample rate is 30 Hz. There are three receivers on the Headstage64, so it's
```
Comment (jonnew) [inline articles/hardware/hs64/ts4231.md:16]: With no occlusions, the maximum achievable sample rate is 30 Hz for each receiver.

**Review by jonnew — APPROVED** [review]
Looks good, please incorporate comments before merge.

**Pull request merged**